File: Codeforces/A_104.java
import java.util.*;
public
class A_104
{
public
    static void main(String[] args)
    {
        Scanner cin = new Scanner(System.in);
        int n = cin.nextInt();
        if (n <= 10 || n >= 22)
        {
            System.out.print(0);
        }
        else if (n == 20)
        {
            System.out.print(15);
        }
        else
        {
            System.out.print(4);
		}
		cin.close();
    }
}

File: Codeforces/A_104.cpp
// solved: 2013-12-24
#include <iostream>
using namespace std;
int main()
{
    int p;
    cin >> p;
    if (p <= 10 || p >= 22)
    {
        cout << 0;
        return 0;
    }
    if (p == 20)
    {
        cout << 4 * 3 + 3;
        return 0;
    }
    else
        cout << 4;
    return 0;
}